tests/e2e/e2e.sh

```diff
@@ -138,6 +138,11 @@ ginkgo -v -nodes=1 --focus="\[ics-e2e\] \[sc\]" ./tests/e2e | tee -a block-vpc-c
 rc1=$?
 echo "Exit status for basic volume test: $rc1"
 
+ginkgo -v -nodes=1 --focus="\[ics-e2e\] \[resize\] \[pv\]" | tee -a block-vpc-csi-volume-resize-ginkgo-log.txt
+rc3=$?
+echo "Exit status for resize volume test: $rc3"
+
+
```

Comment:
Meghna, we need to check rc3 value below and report status accordingly.  [ we have some if condition already for rc1 && rc2 ]